src/Maersk.Test.AutoFixtureExtensions/AutoDataWithCustomizationAttribute.cs

```diff
@@ -25,7 +25,11 @@ public AutoDataWithCustomizationAttribute(params Type[] customizations)
                     customizations.Select(customization =>
                         CustomizationBuilder.CreateCustomization(customization)));
 
-                return new Fixture().Customize(composite);
+                var fixture = new Fixture().Customize(composite);
+
+                fixture.Customize<DateOnly>(c => c.FromFactory<DateTime>(DateOnly.FromDateTime));
```

Comment:
Agree - this should have been fixed inside AutoFixture itself, but until then let's apply the fix here, so that we don't have to rename attributes inside our existing test projects (where `DateOnly` is used).

If/when fixed inside AutoFixture let's come back and delete it here.